cli/UpdateTicketsCommand.php

```diff
@@ -0,0 +1,55 @@
+<?php
+declare( strict_types = 1 );
```

Comment:
Y u ranomly not put the newline between those two things?

![ocd](https://cloud.githubusercontent.com/assets/146040/22591652/eee164ce-ea15-11e6-8dac-d1e548cb9acc.png)